pymeasure/instruments/tdk/tdk_base.py

```diff
@@ -257,29 +243,34 @@ def write(self, command):
 
         Valid frequency values are 18, 23, or 46 Hz. Default value is 18 Hz.
         """,
+        check_set_errors=True,
         validator=strict_discrete_set,
         values=[18, 23, 46]
     )
 
-    source_output = Instrument.control(
+    output_enabled = Instrument.control(
         "OUT?", "OUT %s",
         """Control the output of the power supply.
 
-        Valid values are ``'ON'`` and ``'OFF'``.
+        Valid values are ``True`` and ``False``.
```

Comment:
The, currently recommended, way to describe a property are described here: https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html#docstrings

If you state, that the property is a boolean (e.g. by appending `(bool)` to the introduction sentence), it follows automatically, that the valid values are True or False.